main.go

```diff
@@ -335,15 +336,24 @@ func prepareConfig(parseOpts *cliOptions, conf *ffuf.Config) error {
 		}
 	}
 
-	//Prepare headers
+	//Prepare headers and make canonical
 	for _, v := range parseOpts.headers {
 		hs := strings.SplitN(v, ":", 2)
 		if len(hs) == 2 {
-			conf.Headers[strings.TrimSpace(hs[0])] = strings.TrimSpace(hs[1])
+			// trim and make canonical	
+			var CanonicalHeader string = textproto.CanonicalMIMEHeaderKey (strings.TrimSpace(hs[0]))
```

Comment:
Coding style things like whitespace on this line and the alphabetical import order get automatically fixed when you run `go fmt` in the project directory. We should do that before merging the changes in master.